08-transport.Rmd

```diff
@@ -426,10 +464,13 @@ mat_rail_o = mat_rail[knn_orig, ]
 mat_rail_d = mat_rail[knn_dest, ]
 ```
 
-The final stage is to convert these matrices into meaningful geographic objects, in this case 'multilinestrings' that capture the fact that each stage is a separate line, but part of the same overall trip which can be represented as a simple feature:
+The final stage is to convert these matrices into meaningful geographic objects, in this case simple feature 'multilinestrings' that capture the fact that each stage is a separate line, but part of the same overall trip:
```

Comment:
in fact, you are not really using multilinestrings, so I think we should rephrase (see my code example below also)